docs/docs/authentication-tutorial.md

```diff
@@ -14,24 +14,13 @@ In production, you should use a tested and robust solution to handle the authent
 
 ## Building your Gatsby app
 
-Start by creating a new Gatsby project:
+Start by creating a new Gatsby project using barebones `hello-world` starter:
 
 ```shell
-gatsby new gatsby-auth
+gatsby new gatsby-auth gatsbyjs/gatsby-starter-hello-world
 cd gatsby-auth
 ```
 
-Then, add a more apt title to your newly created site, changing the content of `gatsby-config.js`:
-
-```javascript:title=gatsby-config.js
-module.exports = {
-  siteMetadata: {
-    title: "Gatsby Authentication Tutorial",
-  },
-  plugins: ["gatsby-plugin-react-helmet", "gatsby-plugin-offline"],
-}
-```
-
 Create a new component to hold the links. For now, it will act as a placeholder:
 
 ```jsx:title=src/components/navBar.js
```

Comment:
Nit: but we should probably standardize on a consistent naming pattern.

I'd propose dasherized, e.g. `nav-bar.js`. I'm thinking about this because I've seen some confusion on Twitter re: naming (e.g. when do I use uppercase?) and because of things like gatsbyjs/gatsby#10671

Reply:
See gatsbyjs/gatsby#10691 for more on this

Reply:
good point, I'll adjust it tomorrow

Reply:
done